src/tic_tac_toe/human_player.clj

```diff
@@ -0,0 +1,7 @@
+(ns tic-tac-toe.human-player
+  (:require [tic-tac-toe.board :as board]
+            [tic-tac-toe.ui :as ui]))
+
+(defn move [board marker prompt output]
```

Comment:
I would probably make this a verb since it's a method name (e.g. `makeMove` or `getMove`).